management/README.md

```diff
@@ -4,14 +4,136 @@ This directory contains the configuration needed to setup a management GKE clust
 
 Please refer to the latest [docs](https://master.kubeflow.org/docs/gke/deploy/management-setup/)
 
+## Install Instructions
```

Comment:
[ASK] Do we also need the tools version here?
Ref.: https://www.kubeflow.org/docs/gke/deploy/management-setup/#install-the-required-tools.

Reply:
I was only trying to add instructions that changed here (because it's easier to review instruction change + code together).
I'll merge the instructions to https://www.kubeflow.org/docs/gke/deploy/management-setup/#install-the-required-tools, once we are ready to release.